samples/bluetooth/peripheral_uart/README.rst

```diff
@@ -205,6 +205,43 @@ After programming the sample to your development kit, complete the following ste
 #. Disconnect the device in nRF Connect for Mobile.
    Observe that **LED 2** turns off.
 
+.. _nrf52_computer_testing:
+
+Testing with multiple nRF52 Series devices
+------------------------------------------
+
+If you have an nRF52 Series DK with the :ref:`peripheral_uart` sample and either a dongle or second Nordic Semiconductor development kit that supports Bluetooth Low Energy, you can test the sample on your computer.
+Use the Bluetooth Low Energy app in `nRF Connect for Desktop`_ for testing.
+
+To perform the test, :ref:`connect to the nRF52 Series DK <nrf52_gs_connecting>` and complete the following steps:
+
+1. Connect the dongle or second development kit to a USB port of your computer.
```

Comment:
```suggestion
To perform the test, complete the following steps:

1. Connect to your nRF52 Series DK. 
#. Connect the dongle or second development kit to a USB port of your computer.
```
(connecting is already mentioned at the start of Testing, no need to point to GS here)